src/RolloutScorer/RolloutScorer/CloudTableFactory.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Table;
+using System;
+
+namespace RolloutScorer
+{
+    public interface ICloudTableFactory
+    {
+        CloudTable CreateScoreCardTable(string storageAccountKey);
+        CloudTable CreateDeploymentTable(string uri);
```

Comment:
It seems like this is just "CreateTable".  What else can a mock return, since the return type is "CloudTable".